### PR TITLE
Add Create/Edit Intel module

### LINKED ISSUE #29: Feature Request: Create/Edit Intel
There should be an ability to create intel where it will spawn in a type of intel to collect such as a laptop, notes, documents etc. Moreover, editing an object already to have intel such as a car or a device. 
CreateEdit Intel

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- title, #29

### CODE REVIEW

File: addons/modules/functions/fnc_moduleCreateIntelLocal.sqf [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    QUOTE(_target distance _caller < MAX_DISTANCE),
+    {},
+    {},
+    {
+        params ["_object", "_unit", "", "_args"];
+        _args params ["_title", "_text", "_share", "_delete"];
+
+        private _targets = switch (_share) do {
+            case 0: {
+                call CBA_fnc_players select {side group _x == side _unit}
```
Comment (CreepPork) [inline addons/modules/functions/fnc_moduleCreateIntelLocal.sqf]:
```suggestion
                [] call CBA_fnc_players select {side group _x == side _unit}
```

Comment (CreepPork) [reply addons/modules/functions/fnc_moduleCreateIntelLocal.sqf]: Seems like I was mistaken, the syntax without `[]` is more used.

**Comment by CreepPork** [comment]
Don't merge until conversations in #94 are resolved.

**Comment by CreepPork** [comment]
Is there a notification for Zeus when they pick up intel?

**Comment by CreepPork** [comment]
A notification for the Zeus'es would be appreciated especially when a lot is going on or you are building something else and you need to kickstart a different stage of the mission.

**Comment by mharis001** [comment]
Will leave issue open, can look at further enhancements later in order to merge for 1.5.0.

File: addons/modules/stringtable.xml [hunk 2]
```diff
@@ -1357,6 +1357,9 @@
         <Key ID="STR_ZEN_Modules_ModuleCreateIntel_IntelFound">
             <English>Intel Found</English>
         </Key>
+        <Key ID="STR_ZEN_Modules_ModuleCreateIntel_PlayerFoundIntel">
+            <English>%1 found intel</English>
```
Comment (CreepPork) [inline addons/modules/stringtable.xml]: Intel name would also be helpful.

**Pull request merged**